sf-docker/.env

```diff
@@ -28,6 +28,6 @@ NoSqlDataStorageDisabled=false
 REGISTRY=registry.gitlab.com/innovatrics/smartface/
 
 # Version
-SF_VERSION=latest
-AC_VERSION=latest
-JETSON_VERSION=latest
+SF_VERSION=4.8.0
+AC_VERSION=1.5.0
```

Comment:
full version is 1.5.0.29532

Don't know if its better to have full versions (with build numbers) here, or we use the latest patch version...